documentation/docs/developers/strategies/tutorial.md

```diff
@@ -0,0 +1,92 @@
+
+* `c_did_create_buy_order`: called in response to an `order_created_event`
+* `c_did_create_sell_order`: called in response to an `order_created_event`
+* `c_did_fail_order`: called in response to an `order_filled_event`
+* `c_did_create_sell_order`: called in response to an `order_failed_event`
+* `c_did_cancel_order`: called in response to a `cancelled_event`
+* `c_did_expire_order`: called in response to an `expired_event`
+* `c_did_complete_buy_order`: called in response to an `order_completed_event`
+* `c_did_complete_sell_order`: called in response to an `order_completed_event`
+* `c_did_fail_order_tracker`: called in response to an `order_failed_event`
+* `c_did_cancel_order_tracker`: called in response to an `order_cancelled_event`
+* `c_did_expire_order_tracker`: called in response to an `order_expired_event`
+* `c_did_complete_buy_order_tracker`: called in response to an `order_completed_event`
+* `c_did_complete_sell_order_tracker`: called in response to an `order_completed_event`
+
+#### Market class
+The `market_base` class contains overridable functions that can help get basic information about an exchange that a strategy is operating on, which can include the balance, prices, and order books for any particular asset traded on the exchange. 
+
+* `c_buy`: called when the user wants to place a buy order
+* `c_sell`: called when the user wants to place a sell order
+* `c_cancel`: called when the user wants to place an order cancellation
+* `c_get_balance`: called to get the user’s balance of assets
+* `c_get_available_balance`: called to get the user’s available balance of assets
+* `c_withdraw`: called when the user wants to withdraw assets
+* `c_get_order_book`: called to get the order book for any particular asset
+* `c_get_price`: called to get the price for any particular asset
+* `c_get_order_price_quantum`: called to get the quantum price of an order
+* `c_get_order_size_quantum`: called to get the quantum size of an order
+* `c_quantize_order_price`: called to quantize the price of an order
+* `c_quantize_order_amount`: called to quantize the amount of an order
+* `c_get_fee`: called to get the fee for exchange use
+
+Additionally, this strategy leverages the `Market` class’s `EventReporter` listener object, in order to check if buy/sell orders have been filled or completed, the user has enough balance to place certain orders, and if there are any order cancellations. The `EventLogger` object is also used to log the specific events when they occur. 
+
+#### Exposing new strategy to Hummingbot client
+Make strategy name known to the client by adding name to `hummingbot/client/settings.py` under `STRATEGIES` variable. There should also be a template file that contains config variables and its documentation in the `hummingbot/templates directory`. The naming convention for this yml file is `conf_{strategy name}_TEMPLATE`. 
```

Comment:
If you're specifying a file in the source code, you should include a link to the file in our development branch so people can refer to it quickly.

So, use [hummingbot/client/settings.py](https://github.com/CoinAlpha/hummingbot/blob/development/hummingbot/client/settings.py) instead of `hummingbot/client/settings.py`